includes/features/all_sites.feature

```diff
@@ -36,7 +36,7 @@ Feature: Every page will be checked for the following regions, elements and func
   @safe
   Scenario: Verify that /user page has the appropriate content
     Given I am on "user"
-  #  And I should see the text "Local User Login"
+    Then I should see the text "Local User Login"
```

Comment:
@kmakitan, it turns out this will not work on local installs. On my local install I'm seeing "User account" instead. I wonder if you can find something even more generic to look for.